doc/auto-discovery.md

```diff
@@ -180,111 +138,49 @@ sequenceDiagram
-
-***
-
-##### Manual
-Credentials can be added via EdgeX Secrets:
-
-> **Note:** Replace `<secret-path>` with the name for the new secret, `<username>` with the username,
-> `<password>` with the password, and <mode> with the authentication mode.
-
-```shell
-curl --location --request POST 'http://localhost:59984/api/v2/secret' \
-    --header 'Content-Type: application/json' \
-    --data-raw '
-{
-    "apiVersion":"v2",
-    "path": "<secret-path>",
-    "secretData":[
-        {
-            "key":"username",
-            "value":"<username>"
-        },
-        {
-            "key":"password",
-            "value":"<password>"
-        },
-        {
-            "key":"mode",
-            "value":"<mode>"
-        }
-    ]
-}'
-```
 ## Rediscovery
+The device service is able to rediscover and update devices that have been discovered previously.
+Nothing additional is needed to enable this. It will run whenever the discover call is sent, regardless
+of whether it is a manual or automated call to discover.
 
-The device service is able to rediscover and update devices that have been discovered previously. Nothing additional is needed to enable this. It will run whenever the discover call is sent, regardless of whether it is a manual or automated call to discover.
-
+The following logic to determine if the device is already registered or not.
 
-## Device Status
-The device status goes hand in hand with the rediscovery of the cameras, but goes beyond the scope of just discovery. It is a separate background task running at a specified interval (default 30s) to determine the most accurate operating status of the existing cameras.
-
-### States and Descriptions
-Currently, there are 4 different statuses that a camera can have  
-
-**UpWithAuth**: Can execute commands requiring credentials  
-**UpWithoutAuth**: Can only execute commands that do not require credentials. Usually this means the camera's credentials have not been registered with the service yet, or have been changed.  
-**Reachable**: Can be discovered but no commands can be received.  
-**Unreachable**: Cannot be seen by service at all. Typically, this means that there is a connection issue either physically or with the network.   
-
-### Configuration Options
-- Use `EnableStatusCheck` to enable the device status background service.
-- `CheckStatusInterval` is the interval at which the service will determine the status of each camera.
-
-```toml
-EnableStatusCheck = true
-
-# The interval in seconds at which the service will check the connection of all known cameras and update the device status 
-# A longer interval will mean the service will detect changes in status less quickly
-# Maximum 300s (1 hour)
-CheckStatusInterval = 30
+```mermaid
+%% Note: The node and edge definitions are split up to make it easier to adjust the
+%% links between the various nodes.
+graph TD;
+    %% -------- Node Definitions -------- %%
+    Multicast[Discover Multicast]
```

Comment:
These flowcharts are great for understanding a functionality. For this one I felt it would be better to name as `Discover via multicast` or `Multicast discovery` and the same goes for netscan below.